projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/path-actions/class-simple-delete.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Modules\Optimizations\Page_Cache\Pre_WordPress\Path_Actions;
+
+use Automattic\Jetpack_Boost\Modules\Optimizations\Page_Cache\Pre_WordPress\Filesystem_Utils;
+use Automattic\Jetpack_Boost\Modules\Optimizations\Page_Cache\Pre_WordPress\Logger;
+use SplFileInfo;
+
+class Simple_Delete implements Path_Action {
+	/**
+	 * Delete a file or directory.
+	 *
+	 * @param SplFileInfo $file The file or directory to delete.
+	 * @return int The number of files or directories deleted.
+	 */
+	public function apply_to_path( SplFileInfo $file ) {
```

Comment:
I think this can also return `null` if the directory isn't empty.

Reply:
I'll make it return 0. Other bits of code count the number of files affected.